src/Cms/File.php

```diff
@@ -383,10 +383,23 @@ public function mediaHash(): string
 	/**
 	 * Returns the absolute path to the file in the public media folder
 	 * @internal
+	 *
+	 * @param string|null $thumbName Optional thumb filename
+	 */
+	public function mediaPath(string|null $thumbName = null): string
+	{
+		$thumbName ??= $this->filename();
+
+		return $this->mediaRoot() . '/' . $thumbName;
```

Comment:
`Suggestion:` I would go with `$filename` as this relates not just to images/thumbs but all files. Same for `::mediaUrl()`.